databroker/broker.py

```diff
@@ -0,0 +1,448 @@
+from __future__ import print_function
+import warnings
+import six  # noqa
+from collections import deque, defaultdict
+
+import tzlocal
+import logging
+import numbers
+from .core import (Header, _inspect_descriptor,
+                   get_events as _get_events,
+                   get_table as _get_table,
+                   restream as _restream,
+                   process as _process, Images)
+
+
+# Toolz and CyToolz have identical APIs -- same test suite, docstrings.
+try:
+    from cytoolz.dicttoolz import merge
```

Comment:
This import isn't far off from 'import left_pad'. One single function to do something pretty simple in Python for something that is being prematurely optimized - dictionary merges are for each header, each item in each descriptor. I rather doubt they are a performance bottleneck.

I vote to nuke this dependency.